doc/users/next_whats_new/qt-toolmanager.rst

```diff
@@ -0,0 +1,17 @@
+Added support for QT in new ToolManager
+=======================================
+
+Now it is possible to use the ToolManager with Qt5
+For example
+
+  import matplotlib
+
+  matplotlib.use('QT5AGG')
+  matplotlib.rcParams['toolbar'] = 'toolmanager'
+  import matplotlib.pyplot as plt
+
+  plt.plot([1,2,3])
+  plt.show()
+
+The main example `examples/user_interfaces/toolmanager_sgskip.py` shows more
+details, just adjust the header to use QT instead of GTK3
```

Comment:
As per the call, this probably needs to be explicitly marked as an experimental API so we can change it w/o having to go through deprecation cycles.

Reply:
@jklymak  I don't understand your comment, as soon as you run, it shows a warning about experimental status

Reply:
Sure, and the whats-new should make this explicit as well.